docs/releases/3.1.txt

```diff
@@ -475,12 +475,16 @@ Miscellaneous
   ``django-admin``.
 
 * The ``HttpRequest.is_ajax()`` method is deprecated as it relied on a
-  jQuery-specific way of signifying AJAX calls, while the current usage tends
-  to use the JavaScript `Fetch API
-  <https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API>`_. Depending on
-  your use case, you can either write your own AJAX detection method, or use
-  the new :meth:`.HttpRequest.accepts` method if your code depends on the
-  client ``Accept`` HTTP header.
+  jQuery-specific way of signifying AJAX calls, while current usage tends to
+  use the JavaScript `Fetch API
+  <https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API>`_.
+  Depending on your use case, you can either write your own AJAX detection
+  method, or use the new :meth:`.HttpRequest.accepts` method if your code
+  depends on the client ``Accept`` HTTP header.
+
+  If you are writing your own AJAX detection method, the behavior of
+  ``HttpRequest.is_ajax()`` can be reproduced exactly with
```

Comment:
Maybe ```the behavior of deprecated ``request.is_ajax()`` method can be ... ```

Reply:
I think we don't need to repeat the word "deprecated" within the release note that it's deprecated

Reply:
And what about ``request`` instead of ``HttpRequest``?

Reply:
Done